osu.Game/Online/API/Requests/GetFriendsRequest.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace osu.Game.Online.API.Requests
 {
-    public class GetFriendsRequest : APIRequest<List<APIUser>>
+    public class GetFriendsRequest : APIRequest<List<APIRelation>>
```

Comment:
Note that this works because API-side [the format of the response is changed and is dependent on `x-api-version`](https://github.com/ppy/osu-web/blob/0a5367a4a68a6cdf450eb483251b3cf03b3ac7d2/app/Http/Controllers/FriendsController.php#L45). This means that **at least this single change to this single file must go out before the next release, or the friends displays will be broken.** (Discovered in testing with https://github.com/ppy/osu/pull/30499.)